src/Pimple/Container.php

```diff
@@ -279,4 +294,17 @@ public function register(ServiceProviderInterface $provider, array $values = arr
 
         return $this;
     }
+
+    /**
+     * @param $alias
+     * @param $service
+     *
+     * @return $this
+     */
+    public function alias($alias, $service)
+    {
+        $this->aliases[$alias] = $service;
```

Comment:
Well, this is indeed how aliases are done today, and I don't think we need a method in Pimple itself for that then (which would have to take care of factories and so on)